src/agent/custom-http-agent.ts

```diff
@@ -52,12 +54,18 @@ export class CustomHttpAgent {
   request = async ({
     arg,
     canisterId,
-    method: methodName
+    method: methodName,
+    nonce
   }: Pick<
     // This could have been made agnostic by inlining the types here, but for simplicity and because they are strongly typed, I decided to reuse the interface.
     IcrcCallCanisterRequestParams,
-    'canisterId' | 'method' | 'arg'
+    'canisterId' | 'method' | 'arg' | 'nonce'
```

Comment:
Instead of adding another picked parameter, I propose omitting the only remaining unused one. WDYT?

```
Pick<IcrcCallCanisterRequestParams, 'canisterId' | 'method' | 'arg' | 'nonce'>
// vs suggested
Omit<IcrcCallCanisterRequestParams, 'sender'>
```